main_task.py

```diff
@@ -0,0 +1,37 @@
+import path
+import BLE
+import BLDC
+from commands import Commands
+
+PWM_PIN_FR = 
+PWM_PIN_FL = 
+PWM_PIN_BR = 
+PWM_PIN_BL = 
+DIR_PIN_FR = 
+DIR_PIN_FL = 
+DIR_PIN_BR = 
+DIR_PIN_BL = 
```

Comment:
add values

Reply:
We haven't decided what pins are going where so I don't know what to put there. I left it blank so that when we decide to run it, it will throw an error and not let me. In my opinion it's better than assigning random pins. We don't want to set something that is not meant to be HIGH to HIGH.